addons/l10n_generic_coa/migrations/13.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,35 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+xmlid_renames = [
+    ('l10n_generic_coa.a_capital', 'l10n_generic_coa.capital'),
+    ('l10n_generic_coa.a_dividends', 'l10n_generic_coa.dividends'),
+    ('l10n_generic_coa.a_expense_finance', 'l10n_generic_coa.expense_finance'),
+    ('l10n_generic_coa.a_expense_invest', 'l10n_generic_coa.expense_invest'),
+    ('l10n_generic_coa.a_salary_expense', 'l10n_generic_coa.expense_salary'),
+    ('l10n_generic_coa.cas', 'l10n_generic_coa.non_current_liabilities'),
+    ('l10n_generic_coa.conf_a_expense', 'l10n_generic_coa.expense'),
+    ('l10n_generic_coa.conf_a_pay', 'l10n_generic_coa.payable'),
+    ('l10n_generic_coa.conf_a_recv', 'l10n_generic_coa.receivable'),
+    ('l10n_generic_coa.conf_a_reserve_and_surplus', 'l10n_generic_coa.tax_payable'),
```

Comment:
are you sure about this one? I don't know

Reply:
I checked those and were the same just codes changed.

Reply:
Ok, nice.

Reply:
Hi, @Yasaie 
I did not found where this account used in Odoo, so I`m not sure that it the same.
I think someone could use this account in the case which not related to tax_payable.
If you do not provide an example where Odoo12 use it, I think better remove this line.